enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,270 @@
+## Summary
+
+To enable automated etcd backups of an Openshift cluster from day one after installation, 
+without provided configuration from the user.
+
+## Motivation
+
+The current [automated backup of etcd](https://docs.openshift.com/container-platform/4.15/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#creating-automated-etcd-backups_backup-etcd) of an OpenShift
+cluster relies on user provided configurations. 
+
+To improve the user's experience, this work proposes taking etcd backup using default config from day one, without additional configuration from the user.
+
+### Goals
+
+* Backups should be taken without configuration after cluster installation from day 1.
+* Backups are saved to a default location on each master node's disk using `hostPath` volume.
+* Backups are taken according to a default schedule.
+* Backups are being maintained according to a default retention policy.
+
+### Non-Goals
+
+* Save cluster backups to remote cloud storage (e.g. S3 Bucket).
+  - This could be a future enhancement or extension to the API.
+  - Having backups in an independent storage from OCP cluster is crucial for Disaster Recovery scenarios.
+* Automate cluster restoration.
+* Provide automated backups for non-self hosted architectures like Hypershift.
+
+## Proposal
+
+To enable automated etcd backups of an Openshift cluster from Day 1, without user supplied configuration, this work proposes a default etcd backup option.
+
+The default backup option utilizes a sidecar container within each etcd pod. The container is responsible for maintaining backups within a `hostPath` volume on each master node's disk.
+
+This approach guarantees have backups across all master nodes which is vital in disaster recovery scenarios.
+
+### User Stories
+
+* As a cluster administrator, I would like OCP cluster backup to be taken without configuration from day one.
+* As a cluster administrator, I would like to schedule recurring OCP cluster backups so that I have a recent cluster state to recover from in the event of quorum loss (i.e. losing a majority of control-plane nodes).
+
+### API Extensions
+
+No [API](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions) changes are required.
+
+### Topology Considerations
+TBD
+#### Hypershift / Hosted Control Planes
+TBD
+#### Standalone Clusters
+TBD
+#### Single-node Deployments or MicroShift
+TBD
+### Workflow Description
+TBD
+
+## Design Details
+
+A sidecar container is being deployed within each etcd pod in Openshift cluster in order to take backups according to default configuration using `config.openshift.io/v1alpha1 Backup`.
+
+To distinguish default _no-config_ backups from user configured counterparts, a `config.openshift.io/v1alpha1 Backup` CR's name must be `default`.
+
+Initially, the sidecar container within etcd pods is not running (i.e. disabled). However, creating a `Backup` CR with `name=default` enables the container.
```

Comment:
Why not use the existing mechanisms for backing up which don't rely on sidecars, rather than inventing a new pattern? Is that documented somewhere?

Reply:
> How large is an average OpenShift etcd database, and then from that, how large is a backup expected to be?

it is within range [8,32] GB. 

the backup is way smaller than this, I think `perf&scale` can give us some exact numbers, if needed 

also the `Pruning` functionality makes sure we never run out of space

Reply:
> Why not use the existing mechanisms for backing up which don't rely on sidecars, rather than inventing a new pattern? Is that documented somewhere?

- so the existing mechanism will be also available, this work provides the ability to have `AutomatedBackup` with no configuration from CUs

- Also this allows to have `AutomatedBackup` from Day1, after cluster installation. 

- You can find in `Alternatives` section the result of more than a month of investigation. We settled on this in the end, since it is the most straightforward  and allows to have backups on all master nodes, also it works for `BM` and `SNO` clusters